src/common/arrow/arrow_appender.cpp

```diff
@@ -236,11 +236,7 @@ static void InitializeFunctionPointers(ArrowAppendData &append_data, const Logic
 		break;
 	}
 	case LogicalTypeId::MAP:
-		if (append_data.options.arrow_offset_size == ArrowOffsetSize::LARGE) {
-			InitializeAppenderForType<ArrowMapData<int64_t>>(append_data);
-		} else {
-			InitializeAppenderForType<ArrowMapData<int32_t>>(append_data);
-		}
+		InitializeAppenderForType<ArrowMapData<int32_t>>(append_data);
```

Comment:
Please add a comment describing that there is no LargeMapArray, currently this can easily be mistaken for an oversight